src/Components/ScrivitoExtensions/ChatbotTab/useChatCompletion.js

```diff
@@ -73,6 +73,7 @@ async function startStreaming({
       return fetch(url, {
         ...init,
         headers: cleanHeaders(init?.headers),
+        body: JSON.stringify(JSON.parse(init?.body)),
```

Comment:
* [ ] FYI TS correctly warns that `undefined` cannot be passed to `JSON.parse`.
* [ ] FYI This fix is fragile b/c we rely on a certain correlation of the passed-in body/content-length.

Reply:
Did I understand correctly that this a hack, to work around a bug in our AI backend? Why don't we fix our AI backend properly, instead of patching the OpenAI client on the frontend? Or what did I miss here?

Reply:
This has already been removed in #4

Reply:
Nice, good to know 👍